File: Replace_Parameter_with_Method_Call.py
# After
basePrice = quantity * itemPrice
seasonalDiscount = store.getSeasonalDiscount()
fees = store.getFees()
finalPrice = discountedPrice(basePrice, seasonalDiscount, fees)

# Refactoring
basePrice = quantity * itemPrice
finalPrice = discountedPrice(basePrice, store)

File: Replace_Parameter_with_Method_Call.js
// After
var basePrice = quantity * itemPrice;
var seasonalDiscount = store.getSeasonalDiscount();
var fees = store.getFees();
var finalPrice = discountedPrice(basePrice, seasonalDiscount, fees);

// Refactoring
var basePrice = quantity * itemPrice;
var finalPrice = discountedPrice(basePrice, store);
